src/network-config.js

```diff
@@ -1,26 +1,31 @@
 import {
-  getLocalChainId,
+  getDefaultEthNode,
   getEnsRegistryAddress,
   getFortmaticApiKey,
+  getLocalChainId,
   getPortisDappId,
 } from './local-settings'
 
-const localEnsRegistryAddress = getEnsRegistryAddress()
 const fortmaticApiKey = getFortmaticApiKey()
 const portisDappId = getPortisDappId()
 
-export const networkConfigs = {
-  main: {
+const providedDefaultEthNode = getDefaultEthNode()
+const providedEnsRegistryAddress = getEnsRegistryAddress()
+
+const networkConfigurations = {
+  // Mainnet
+  mainnet: {
     addresses: {
       ensRegistry:
-        localEnsRegistryAddress || '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
+        providedEnsRegistryAddress ||
+        '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
     },
-    nodes: {
-      defaultEth: 'wss://mainnet.eth.aragon.network/ws',
+    endpoints: {
+      read: providedDefaultEthNode || 'wss://mainnet.eth.aragon.network/ws',
     },
-    settings: {
+    environment: {
       chainId: 1,
-      name: 'Mainnet',
+      name: 'Ethereum',
```

Comment:
I dig this new naming a lot; but maybe it can be clearer that this is the Ethereum _mainnet_ network?